contrib/intel/jenkins/tests.py

```diff
@@ -651,3 +651,78 @@ def execute_cmd(self, oneccl_test):
                         outputcmd = shlex.split(command)
                         common.run_command(outputcmd)
 
+class OneCCLTestsGPU(Test):
+
+    def __init__(self, jobname, buildno, testname, core_prov, fabric,
+                 hosts, ofi_build_mode, util_prov=None):
+        super().__init__(jobname, buildno, testname, core_prov, fabric,
+                         hosts, ofi_build_mode, util_prov)
+
+        self.n = 2
+        self.ppn = 4
+        self.oneccl_path = '{}/oneccl_gpu/build'.format(self.ci_middlewares_path)
+
+        self.examples_tests = {
+                                  'sycl_allgatherv_custom_usm_test',
+                                  'sycl_allgatherv_inplace_test',
+                                  'sycl_allgatherv_inplace_usm_test',
+                                  'sycl_allgatherv_test',
+                                  'sycl_allgatherv_usm_test',
+                                  'sycl_allreduce_inplace_usm_test',
+                                  'sycl_allreduce_test',
+                                  'sycl_allreduce_usm_test',
+                                  'sycl_alltoall_test',
+                                  'sycl_alltoall_usm_test',
+                                  'sycl_alltoallv_test',
+                                  'sycl_alltoallv_usm_test',
+                                  'sycl_broadcast_test',
+                                  'sycl_broadcast_usm_test',
+                                  'sycl_reduce_inplace_usm_test',
+                                  'sycl_reduce_scatter_test',
+                                  'sycl_reduce_scatter_usm_test',
+                                  'sycl_reduce_test',
+                                  'sycl_reduce_usm_test'
+                              }
+        self.functional_tests = {
+                                    'allgatherv_test',
+                                    'alltoall_test',
+                                    'alltoallv_test',
+                                    'bcast_test',
+                                    'reduce_scatter_test',
+                                    'reduce_test'
+                                }
+
+    @property
+    def cmd(self):
+        return "{}/run_oneccl_gpu.sh ".format(ci_site_config.testpath)
+
+    def options(self, oneccl_test_gpu):
+        opts = "-n {n} ".format(n=self.n)
+        opts += "-ppn {ppn} ".format(ppn=self.ppn)
+        opts += "-hosts {server},{client} ".format(server=self.server,
```

Comment:
Do these -hosts need to contain server-interface,client-interface instead of just self.server,self.client?

If they need it: the other tests use common.get_node_name(self.server, self.nw_interface_map) to format this for you.